packages/angular-playground/src/index.html

```diff
@@ -24,21 +24,18 @@
     <title>Angular Playground</title>
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.10/angular.js"></script>
     <script src="app.js"></script>
-    <style>
-      orion-button {
-        margin:1em 0;
-      }
-    </style>
   </head>
   <body ng-app="app">
     <h1>Angular Playground</h1>
     <hr />
     <div ng-controller="Controller as ctrl">
       <h2>Buttons</h2>
       <a href="#" ng-click="ctrl.toggleDisability()">Toggle Button Disability</a> |
-      <a href="#" ng-click="ctrl.embiggen()">Make all buttons big</a>
+      <orion-select on-change="ctrl.setSize(event)" options="sizes" ng-model="buttonSize"></orion-select>
```

Comment:
What is the purpose of ng-model here?

I'd like to avoid using two-way binding at all.